package.json

```diff
@@ -10,7 +10,7 @@
     "inkdrop": "^5.x"
   },
   "dependencies": {
-    "mermaid": "8.13.10"
+    "mermaid": "^9.1.1"
```

Comment:
can you fix the version to the exact "9.1.1" to avoid unexpected issues?

Reply:
Done